### PR TITLE
Add Chapter 1.C rules to "all the rules we know"

### PR DESCRIPTION
**Pull request by llewelld**
Tidies up the previous sections and adds new rules and results for 1.C.

### CODE REVIEW

**Comment by llewelld** [comment]
See the [PDF preview](https://github.com/llewelld/hut23-linear-algebra/blob/gh-action-result/pdf-files/reference/all-the-rules-we-know.pdf) of the changes for an idea of the result.

File: reference/all-the-rules-we-know.tex [hunk 1]
```diff
@@ -169,29 +173,29 @@ \section*{Chapter 1.A}
 For $(x_1, \ldots, x_n ) \in \F^n$ and $j \in \{ 1, \ldots, n \}$, we say that $x_j$ is the $j^{\text{th}}$ \defn{coordinate} of $(x_1, \ldots, x_n )$.
 \end{definition}
 
-\begin{definition}{1.12}[addition in $F^n$]
+\begin{definition}{1.13}[addition in $F^n$]
```
Comment (triangle-man) [inline reference/all-the-rules-we-know.tex:176]:
```suggestion
\begin{definition}{1.13}[addition in $\F^n$]
```

File: reference/all-the-rules-we-know.tex [hunk 2]
```diff
@@ -169,29 +173,29 @@ \section*{Chapter 1.A}
...
 \defn{Addition} in $F^n$ is defined by adding corresponding coordinates:
```
Comment (triangle-man) [inline reference/all-the-rules-we-know.tex:177]:
```suggestion
\defn{Addition} in $\F^n$ is defined by adding corresponding coordinates:
```

File: reference/all-the-rules-we-know.tex [hunk 3]
```diff
@@ -169,29 +173,29 @@ \section*{Chapter 1.A}
...
-\begin{definition}{1.14}[$0$]
+\begin{definition}{1.15}[$0$]
 Let $0$ denote the list of length $n$ whose coorinates are all 0:
 $$
 (0, \ldots, 0 ).
 $$
 \end{definition}
 
-\begin{definition}{1.16}[additive inverse in $F^n$]
+\begin{definition}{1.17}[additive inverse in $F^n$]
```
Comment (triangle-man) [inline reference/all-the-rules-we-know.tex:190]:
```suggestion
\begin{definition}{1.17}[additive inverse in $\F^n$]
```

File: reference/all-the-rules-we-know.tex [hunk 4]
```diff
@@ -169,29 +173,29 @@ \section*{Chapter 1.A}
...
 For $x \in \F^n$, the \defn{additive inverse} of $x$, denoted $-x$, is the vector $-x \in \F^n$ such that
 $$
 x + (-x) = 0.
 $$
 In other words, if $x = (x_1, \ldots, x_n)$, then $-x = (-x_1, \ldots, -x_n)$.
 \end{definition}
 
-\begin{definition}{1.17}[scalar multiplication in $F^n$]
+\begin{definition}{1.18}[scalar multiplication in $F^n$]
```
Comment (triangle-man) [inline reference/all-the-rules-we-know.tex:198]:
```suggestion
\begin{definition}{1.18}[scalar multiplication in $\F^n$]
```

File: reference/all-the-rules-we-know.tex [hunk 5]
```diff
@@ -270,40 +274,105 @@ \section*{Chapter 1.B}
...
 $(-1)v = -v$ for every $v \in V$.
 \end{result}
 
 \newpage
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section*{Chapter 1.C}
+
+\begin{definition}{1.33}[subspace]
+A subset $U$ of $V$ is called a \defn{subspace} of $V$ if $U$ is also a vector space with the same additive identity, addition, and scalar multilication as on $V$.
```
Comment (triangle-man) [inline reference/all-the-rules-we-know.tex:319]:
```suggestion
A subset $U$ of $V$ is called a \defn{subspace} of $V$ if $U$ is also a vector space with the same additive identity, addition, and scalar multiplication as on $V$.
```

File: reference/all-the-rules-we-know.tex [hunk 6]
```diff
@@ -270,40 +274,105 @@ \section*{Chapter 1.B}
...
+$$
+V_1 + \cdots + V_m = \{v_1 + \cdots + v_m \setsep v_1 \in V_1, \ldots, v_m \in V_m \}.
+$$
+\end{definition}
+
+\begin{definition}{1.41}[direct sum, $\oplus$]
+Suppose $V_1, \ldots, V_m$ are subspaces of $V$.
+
+\begin{enumerate}
+\item The sum $V_1 + \cdots + V_m$ is called a \defn{direct sum} if each element of $V_1 + \cdots + V_m$ can be written in only one way as a sum $V_1 + \cdots + v_m$, where each $V_k \in V_k$.
```
Comment (triangle-man) [inline reference/all-the-rules-we-know.tex:333]:
```suggestion
\item The sum $V_1 + \cdots + V_m$ is called a \defn{direct sum} if each element of $V_1 + \cdots + V_m$ can be written in only one way as a sum $v_1 + \cdots + v_m$, where each $v_k \in V_k$.
```

File: reference/all-the-rules-we-know.tex (hunk 1 above)
Comment (llewelld) [reply reference/all-the-rules-we-know.tex:176]: Nice catch; thank you!

File: reference/all-the-rules-we-know.tex (hunk 2 above)
Comment (llewelld) [reply reference/all-the-rules-we-know.tex:177]: And again; thank you.

File: reference/all-the-rules-we-know.tex (hunk 3 above)
Comment (llewelld) [reply reference/all-the-rules-we-know.tex:190]: I'm beginning to see a pattern here.

File: reference/all-the-rules-we-know.tex (hunk 5 above)
Comment (llewelld) [reply reference/all-the-rules-we-know.tex:319]: Thank you. I should have spellchecked.

**Comment by llewelld** [comment]
Thanks for the changes @triangle-man; much appreciated. I've incorporated them all in, you can see the result [in this preview](https://github.com/llewelld/hut23-linear-algebra/blob/gh-action-result/pdf-files/reference/all-the-rules-we-know.pdf).

**Comment by llewelld** [comment]
Thanks for the fixes @triangle-man!

**Pull request merged**